ci-operator/step-registry/kueue-operator/test/perfscale/kueue-operator-test-perfscale-chain.yaml

```diff
@@ -1,19 +1,51 @@
 chain:
```

Comment:
Why do we need to change this job?

Maybe perf-scale team can look this over and see this breaks their jobs.
cc @rsevilla87 @avasilevskii